package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "ecobalyse-data",
+  "version": "0.1.0",
+  "description": "Accélérer l'affichage environnemental de la filière textile française",
+  "author": "Ecobalyse <contact@beta.gouv.fr>",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "db:validate": "npx ajv validate --spec=draft2019 -c ajv-formats -s tests/processes-schema.json -d 'public/data/**/processes**.json'",
+    "export:all": "npm run export:food && npm run export:textile && npm run export:object && npm run fix:all",
+    "export:food": "uv run python food/export.py",
+    "export:object": "uv run python object/export.py",
+    "export:textile": "uv run python textile/export.py",
+    "import:all": "npm run import:food && npm run import:ecoinvent && npm run import:method && npm run import:create_activities && npm run import:sync_datapackages",
+    "import:create_activities": "uv run python create_activities.py",
+    "import:ecoinvent": "uv run python import_ecoinvent.py",
+    "import:food": "uv run python import_food.py",
+    "import:method": "uv run python import_method.py",
+    "import:sync_datapackages": "uv run python common/sync_datapackages.py",
+    "jupyter:lab": "uv run jupyter lab",
+    "lint:prettier": "prettier --config .prettierrc --check",
+    "lint:prettier:all": "npm run lint:prettier -- .",
+    "lint:ruff:all": "npm run lint:ruff:check && npm run lint:ruff:format",
+    "lint:ruff:check": "uv run ruff check --force-exclude --extend-select I",
+    "lint:ruff:format": "npm run fix:ruff:format -- --check",
+    "lint:all": "npm run lint:prettier:all && npm run lint:ruff:all && npm run lint:openapi",
+    "fix:prettier": "npm run lint:prettier -- --write",
+    "fix:prettier:all": "npm run lint:prettier -- --write .",
```

Comment:
Note that this won't format files behind `ECOBALYSE_DATA_DIR` when set outside of current repo